app/lib/default_error_messages.rb

```diff
@@ -30,5 +30,11 @@ def self.generate!
       source: :internal,
       expose: false
     )
+    EfileError.find_or_create_by!(
+      code: "BANK-DETAILS",
+      message: "Some of the provided bank account details are incorrect or absent.",
+      source: :client,
```

Comment:
we don't have a source 'client' already. We have internal, usps, and irs. This doesn't truly feel like any of those, but I don't know if this makes sense either. Any suggestions?